pkg/generators/models/models.go

```diff
@@ -59,6 +59,8 @@ type Model struct {
 	Properties []PropSpec
 	// ConvertSpecs contains a list of convert functions for this model
 	ConvertSpecs []ConvertSpec
+	// Discriminator contains the optional oneOf discriminator
+	Discriminator Discriminator
```

Comment:
I thought to put the discriminator data into the convert spec, but the OpenAPI spec does not validate that the mapping is one-to-one, so i kept the mapping as a separate piece of data inside the Discriminator